src/main/java/org/mqttbee/api/mqtt/MqttClientSslConfigBuilder.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 The MQTT Bee project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mqttbee.api.mqtt;
+
+import org.mqttbee.annotations.NotNull;
+import org.mqttbee.annotations.Nullable;
+import org.mqttbee.mqtt.MqttClientSslConfigImpl;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.TrustManagerFactory;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.util.concurrent.TimeUnit;
+
+public class MqttClientSslConfigBuilder {
+
+    private KeyStore keyStore = null;
+    private String keyStorePassword = "";
+    private KeyStore trustStore = null;
+    private long handshakeTimeoutMs = MqttClientSslConfig.DEFAULT_HANDSHAKE_TIMEOUT_MS;
+
+    @NotNull
+    public MqttClientSslConfigBuilder keyStore(@Nullable final KeyStore keyStore) {
```

Comment:
to be consistent with our other builders the methods should start "with...", e.g. "withKeyStore"

Reply:
You are right, to be consistent we must add "with" or "using" prefixes. I would generally prefer to remove them. Probably we could discuss this briefly in the jour fix.

Reply:
Ok, good point for a short discussion.
I would add the prefixes here and remove them from all builders with a seperate PR if we choose to remove the prefixes.